torchbench.py

```diff
@@ -552,6 +551,9 @@ def main():
         "--speedup-trt", action="store_true", help=help(speedup_experiment_trt)
     )
     group.add_argument("--python-key", action="store_true")
+    group.add_argument(
+        "--speedup-fx2trt", action="store_true", help=help(speedup_experiment)
```

Comment:
I think this `help=` string will be for the wrong thing.

Reply:
Thanks for pointing out. I created another similar function speedup_experiment_fx2trt